QueryProfiler/Profile/ProfileScheme.cs

```diff
@@ -7,18 +7,10 @@ namespace QueryProfiler
 {
    public class ProfileScheme
     {     // Operators
-        public int SearchCounter { get; set; }
-        public int ExtendCounter { get; set; }
-        public int JoinCounter { get; set; }

```

Comment:
join, lookup, in are the more effecting efficiency operators, those are the ones which are interesting
think about what each of the operator is actually doing, you will get by yourself to the conclusion that joining two (might be) big tables is much heavier than checking if a string contains sub string for example

additional operators you should consider adding: union, mv-expand